tools/src/main/java/org/apache/kafka/tools/VerifiableConsumer.java

```diff
@@ -649,8 +648,12 @@ public static VerifiableConsumer createFromArgs(ArgumentParser parser, String[]
             if (groupRemoteAssignor != null)
                 consumerProps.put(ConsumerConfig.GROUP_REMOTE_ASSIGNOR_CONFIG, groupRemoteAssignor);
         } else {
-            // This means we're using the old consumer group protocol.
+            // This means we're using the CLASSIC consumer group protocol.
             consumerProps.put(ConsumerConfig.PARTITION_ASSIGNMENT_STRATEGY_CONFIG, res.getString("assignmentStrategy"));
+            Integer sessionTimeout = res.getInt("sessionTimeout");
```

Comment:
If this tool does not offer the default value, it is fine to throw exception if user does set the `--session-timeout`

Reply:
> it is fine to throw exception if user does set the --session-timeout

Pardon me, did you mean throw exception if user set session timeout when using CONSUMER group.protocol ? Not sure if I understand correctly 🙏

Reply:
yes, we should honor users' configs to let him encounter the error.